src/UI/Actions.hs

```diff
@@ -388,8 +388,22 @@ removeTags ts =
 reloadList :: Action 'BrowseThreads AppState
 reloadList = Action "reload list of threads" applySearch
 
+selectNextUnread :: Action 'BrowseMail AppState
```

Comment:
I've currently only implemented it for `'BrowseMail` and I think that's enough for now. However I can see that we want to change it once we have #112 implemented.